int16 update_mode
---
bool result